cilium-dbg/cmd/policy_validate.go

```diff
@@ -22,22 +22,17 @@ var policyValidateCmd = &cobra.Command{
 		if ruleList, err := loadPolicy(path); err != nil {
 			Fatalf("Validation of policy %s has failed: %s\n", path, err)
 		} else {
-			for _, r := range ruleList {
-				if err := r.Sanitize(); err != nil {
-					Fatalf("Validation of policy %s has failed: %s\n", path, err)
-				}
+			jsonPolicy, err := json.MarshalIndent(ruleList, "", "  ")
+			if err != nil {
+				Fatalf("Cannot marshal policy %s: %s\n", path, err)
 			}
-			if policyVerbose {
+			if resp, err := client.PolicyValidate(string(jsonPolicy)); err != nil {
```

Comment:
I suspect the reason CI is failing is because of the swap from `Sanitize()` -> `PolicyValidate()` here, introducing a hard dependency on the server.

Reply:
yeah true, it's not a full server anyway